tests/expected/htmlbars/htmlbars-test-helpers.js

```diff
@@ -1 +1 @@
-define("htmlbars-test-helpers",  ["exports"],  function(__exports__) {    "use strict";    function foo() {    }    __exports__.foo = foo;function bar() {    }    __exports__.bar = bar;  });
+define('htmlbars-test-helpers', ['exports'], function (exports) {	'use strict';	exports.foo = foo;	exports.bar = bar;	function foo() {}	function bar() {}});
```

Comment:
This assertion has been removed (just now in emberjs-build 0.1.0), can you remove this fixture file?